validators/hardware-checklist.adoc

```diff
@@ -0,0 +1,52 @@
+= Mainnet Hardware Checklist
+:icons: font
+:toc: macro
+
+ifdef::env-github[]
+
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+
+toc::[]
+
+endif::[]
+
+[options="interactive"]
+.SKALE Validator
+* [ ] A wallet (recommended: Ledger or SGXWallet; or others)
```

Comment:
they get confused about the SGX wallet can be also used as a Validator wallet. Can we add here this will be used by Validator-CLI?